### DISCUSSION TITLE
Meter update time

### DISCUSSION THREAD

**Discussion by ondras76** [wmbusmeters/wmbusmeters#1041] [General]
Hello, can you help me pls. I was try find here, but I didn't get an answer. Is any chance setup interval of metering? Thank you, Ondrej

**Comment by BIBOLV** [comment]
Hi, if you are asking if you can influence t,c wireless mbus meter reading reporting interval from wmbusmteres side, then answer is no - that depends on meter configuration. It should be possible to change reporting interval in meter, but mostly that can do only manufacturer or distributor of meters.

**Reply by ondras76** [reply]
hello, thank you for your answer. Propably you'r right. Only what is strange is, if I restart the meter read the actualy state almost immediately. But next read id is about forty minutes every hour.

**Reply by BIBOLV** [reply]
What do you mean by restarting meter? Usually it is not possible by design.

**Reply by ondras76** [reply]
I mean if I restart pc, the first metering is very soon.

**Reply by BIBOLV** [reply]
What meter do you have?

**Reply by ondras76** [reply]
Sensus iPerl.

**Reply by BIBOLV** [reply]
Might be that it only appears that you receive it faster - try to do restart right after receiving telegram and check that next telegram is after usual interval.

**Reply by marianri** [reply]
I have a meter, Techem Radio4, it transmits the index once every 35 seconds, in the description of the equipment it is said that it can only send water consumption twice a month, can Wmbusmeters set the time between two transmissions? I know that the T1 protocol is unidirectional, but I still don't understand the counter's behavior.
